sycl/cmake/modules/FetchOpenCL.cmake

```diff
@@ -0,0 +1,87 @@
+# Finds or fetches OpenCL Headers and the ICD loader.
+if(TARGET OpenCL-Headers)
+  return()
+endif()
+
+# Repo URLs
+
+set(OCL_HEADERS_REPO
+  "https://github.com/KhronosGroup/OpenCL-Headers.git")
+set(OCL_LOADER_REPO
+  "https://github.com/KhronosGroup/OpenCL-ICD-Loader.git")
```

Comment:
I expect SYCL to have no dependencies on OpenCL. We intentionally removed this code from the SYCL cmake in the past (see https://github.com/intel/llvm/commit/66919a24234f9e1f4673bfb42ce6b608f811b91a) because SYCL links with the unified runtime.
The unified runtime handles dependencies on HW specific APIs like CUDA, HIP, OpenCL/L0, etc.

Reply:
UR should be a better place because it manages low-level APIs.